src/components/modal/modal.tsx

```diff
@@ -83,8 +82,7 @@ export const Modal = ({
 			showModal={showModal}
 			background={background}
 			data-testid="modal-container"
-			setShowModal={setShowModal}
-		>
+			setShowModal={setShowModal}>
```

Comment:
the closing of the component should be on the next line as before - can you check in your settings?